### ISSUE TITLE
Back-to-top button / navigation.top overlays search bar

### ISSUE THREAD

**Issue by morido** [squidfunk/mkdocs-material#2862]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

New back-to-top button (Material for mkdocs >= 7.1.10) overlays search bar when search bar is focused.

### Expected behaviour

This is the behaviour on the Material for MKDocs website which I'd consider correct:

![firefox_working](https://user-images.githubusercontent.com/26249/126862089-ffcf15de-3759-475a-9af2-f99c1bcee01c.png)

### Actual behaviour

This is the behaviour after setting up a new site in a venv on my local machine:

![firefox_notworking](https://user-images.githubusercontent.com/26249/126862091-e4a85eaa-1569-4beb-a3cc-b7ea9543b951.png)

The "back to top" button overlays the search bar for some odd reason.

### Steps to reproduce

1. Scroll such that "Back to top button" appears
2. Focus search bar

### Package versions

- Python: Python 3.6.9
- MkDocs: mkdocs, version 1.2.2
- Material: Version: 7.2.0

### Configuration

```yaml
site_name: Donald Duck's site

theme:
  name: material  
  features:
    - navigation.top
```

### System information

- Operating system: Ubuntu 18.04.5 LTS
- Browser: Firefox 90.0 / Chromium 91.0

**Comment by squidfunk** [comment]
Thanks for reporting. I already fixed this in Insiders but forgot to backport it, I'll issue a fix right away.

**Comment by squidfunk** [comment]
Fixed in b740dbc97.

**Comment by squidfunk** [comment]
Released as part of 7.2.1.
